getting-started/git-boot-camp.rst

```diff
@@ -439,16 +439,25 @@ Both of these tools will configure a remote URL for the branch, so you can
 ``git push`` if the pull request author checked "Allow edits from maintainers"
 when creating the pull request.
 
-If you don't have GitHub CLI or hub installed, you can set up a git alias. On
-Unix and macOS::
+If you don't have GitHub CLI or hub installed, you can set up a git alias:
 
-   $ git config --global alias.pr '!sh -c "git fetch upstream pull/${1}/head:pr_${1} && git checkout pr_${1}" -'
+.. tab:: Unix
```

Comment:
I think we can combine these into a single tab:

```suggestion
.. tab:: Unix/macOS
```

Reply:
Makes sense to me!